src/main/java/cubicchunks/worldgen/GeneratorPipeline.java

```diff
@@ -25,31 +25,33 @@
 
 import cubicchunks.CubicChunks;
 import cubicchunks.server.ServerCubeCache;
-import cubicchunks.util.AddressTools;
-import cubicchunks.util.Progress;
+import cubicchunks.util.CubeCoords;
 import cubicchunks.util.processor.CubeProcessor;
-import cubicchunks.util.processor.QueueProcessor;
 import cubicchunks.world.cube.Cube;
 import cubicchunks.world.dependency.CubeDependency;
 import cubicchunks.worldgen.dependency.DependentCube;
 import cubicchunks.worldgen.dependency.DependentCubeManager;
 
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Queue;
 
 public class GeneratorPipeline {
 
-	private static final int TickBudget = 40; // ms. There are only 50 ms per tick
+	private static final int MAX_CUBES_PER_TICK = 500;
```

Comment:
this is way too high. The goal of this is to make the server still work relatively smoothly even while generating chunks.